dashboard/src/app/modules/authentication/modules/change-password/components/change-password/change-password.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+import { AuthenticationService } from '~/core/services/authentication/authentication.service';
+import { password } from '~/shared/config/validators';
+
+@Component({
+  selector: 'app-change-password',
+  templateUrl: './change-password.component.html',
+  styleUrls: ['./change-password.component.scss'],
+})
+export class ChangePasswordComponent implements OnInit {
+  public changePasswordForm: FormGroup;
+
+  constructor(private fb: FormBuilder, private authentication: AuthenticationService) {}
+
+  ngOnInit(): void {
+    this.initProfilForm();
+  }
+
+  get controls() {
+    return this.changePasswordForm.controls;
+  }
+  onChangePassword(): void {
+    this.authentication.changePassword(
+      this.changePasswordForm.value.oldPassword,
+      this.changePasswordForm.value.newPassword,
```

Comment:
ça y'a moyen de le factoriser en juste `this.changePasswordForm.value`

Vu que j'ai besoin de la branche je vais quand même merge, puis c'est pas quelque chose de bloquant